pandas/core/arrays/sparse/accessor.py

```diff
@@ -369,10 +369,9 @@ def to_dense(self) -> DataFrame:
         1  1
         2  0
         """
-        from pandas import DataFrame
-
+        constructor = self._parent._constructor
```

Comment:
Nit: Could you inline this on the `return` line?

Reply:
Sure, done.